src/views/AddItem.jsx

```diff
@@ -4,11 +4,43 @@ import React, { useState } from 'react';
 //LOCAL IMPORTS
 import { addItem } from '../api/firebase.js';
 
-export function AddItem({ listToken }) {
+export function AddItem({ listToken, data }) {
 	// SET STATES
 	const [itemName, setItemName] = useState('');
 	const [days, setDays] = useState(7);
 	const [status, setStatus] = useState(null);
+	// NON RegEx VERSION FOR CLEANING INPUT
```

Comment:
I would remove this commented out part since your other method is the one that's being merged. Trying out both was a great idea!